biz.aQute.bndlib/src/aQute/bnd/service/resource/SupportingResource.java

```diff
@@ -7,13 +7,13 @@
 import org.osgi.resource.Resource;
 
 /**
- * The SupportingResource interface represents a resource that requires other
- * resources that are needed to resolve the primary resource. It was introduced
+ * The SupportingResource interface represents a resource that has multiple
```

Comment:
> I disagree that the synthetic use case is not appropriate. So please adjust the comments and do not restrict the use here to your particular use case.

I'm surprised that you read this into the changes. I tried to change the documentation to be less opinionated about the structure, not more. The previous wording more or less restricted SupportingResource to being the synthetic use case, rather than SupportingResource being a general mechanism for having one resource encapsulate several others.

Reply:
well, the original idea was to be exactly as it was word, supporting resources. As said, I am open to changing the name and semantics but to me the synthetic use case is too interesting to not be supported by this. Clearly the semantics are clear that the primary resource needs to be expanded in the case of a repository and that works for both cases. However, you're capability copying is clearly _not_ working for my use case.

So we have now two case, crutches, and variants. Not sure these are the only ones. Needs some more thinking.

Anyway, the comment was about the primary reason it was selected, which still is the synthetic case. It would be nicer to just add the second use case to the comments as well instead of removing the original.